internal/deviations/deviations.go

```diff
@@ -935,4 +935,25 @@ func SkipBgpSendCommunityType(dut *ondatra.DUTDevice) bool {
 // BgpActionsSetCommunityMethodUnsupported return true if BGP actions set-community method is unsupported
 func BgpActionsSetCommunityMethodUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetBgpActionsSetCommunityMethodUnsupported()
+
+}
+
+// IPv4StaticRouteWithIPv6NextHopUnsupported unsupported ipv4 with ipv6 nexthop
+func IPv4StaticRouteWithIPv6NextHopUnsupported(dut *ondatra.DUTDevice) bool {
+	return lookupDUTDeviations(dut).GetIpv4StaticRouteWithIpv6NhUnsupported()
+}
+
+// IPv6StaticRouteWithIPv4NextHopUnsupported unsported ipv4 with ipv4 next-hop
```

Comment:
typos: 
unsupported
ipv_6_ with ipv4 next-hop

Reply:
i belive your talking about this line `unsported ipv4 with ipv4 next-hop` this is changed to `unsported ipv6 with ipv4 nexthop`

Reply:
yes